pjsip-apps/src/pjsua/pjsua_app.c

```diff
@@ -696,11 +744,37 @@ static void on_buddy_evsub_state(pjsua_buddy_id buddy_id,
 
 }
 
+static void on_buddy_evsub_blf_state(pjsua_buddy_id buddy_id,
+                 pjsip_evsub *sub,
+                 pjsip_event *event)
+{
+    char event_info[80];
+
+    PJ_UNUSED_ARG(sub);
+
+    event_info[0] = '\0';
+
+    if (event->type == PJSIP_EVENT_TSX_STATE &&
```

Comment:
`event` might be NULL

Reply:
I would disagree.
This approach is the same as used in the static void on_buddy_evsub_state() just 25 lines above.